scripts/run-release-tests.sh

```diff
@@ -20,3 +20,8 @@ export RUN_KOPS_TEST=true
 echo "Running KOPS test"
 ./scripts/run-integration-tests.sh
 unset RUN_KOPS_TEST
+
+export RUN_CALICO_TEST=true
+echo "Running calico test"
+./scripts/run-integration-tests.sh
+unset RUN_CALICO_TEST
```

Comment:
nit: a newline?

Reply:
Will take care of this in the next commit.